tests/00-check-cmdref.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
```

Comment:
Can we run this as part of the build instead or is it too expensive?

Reply:
We can do that, but I still, want the runtime check which can prevent someone from forgetting to check-in files.

Reply:
OK, let's do both then